src/Tests/dotnet-watch.Tests/Utilities/WatchableApp.cs

```diff
@@ -76,7 +78,7 @@ public void Start(IEnumerable<string> arguments, [CallerMemberName] string name
 
             var commandSpec = new DotnetCommand(_logger, args.ToArray())
             {
-                WorkingDirectory = SourceDirectory,
+                WorkingDirectory = WorkingDirectory ?? SourceDirectory,
```

Comment:
```suggestion
                WorkingDirectory ??= SourceDirectory,
```

Reply:
That wouldn't work - we're assigning the `WorkingDirectory` property from the current instance to the `DotNetCommand` instance.